src/server/enterprise/cmds/cmds.go

```diff
@@ -124,11 +125,17 @@ func RegisterCmd() *cobra.Command {
 			}
 			defer ec.Close()
 
+			if pachdEntAddr == "" {
+				pachdEntAddr = pachdAddr
```

Comment:
Would it make sense to use the address from `c` (the admin's connection to the pachd) rather than defaulting to the enterprise address? We could actually use that as the default for the enterprise connection too? The cases I'm imagining are either the pachd is on the internet - both the enterprise server and the admin can use the internet-facing address - or the users and the enterprise server have to use different addresses because users are port-forwarding or using a VPN or something similar.

Reply:
that makes a lot of sense :)